Packs/MicrosoftExchangeOnPremise/Integrations/EWSv2/EWSv2.py

```diff
@@ -752,6 +752,23 @@ def send_email_to_mailbox(account, to, subject, body, body_type, bcc, cc, reply_
     return m
 
 
+def get_message_for_body_type(body, body_type, html_body):
+    """
+    Compatibility with Data Collection - where body_type is not provided, we will use the html_body if it exists.
+    Compatibility with 'send-mail' command - where body_type should be provided, we will use the body_type to decide.
+    Args:
+        body_type: type of the body. Can be 'html' or 'text' or None.
+        body: plain text body.
+        html_body: HTML formatted content (body) of the email to be sent.
+
+    Returns:
+        Body: the body of the message.
+    """
+    if body_type is None:
+        return HTMLBody(html_body) if html_body else Body(body)
+    return HTMLBody(html_body) if body_type.lower() == 'html' and html_body else Body(body)
```

Comment:
for readability, let's make it 
```suggestion
    if body_type.lower() == 'html' and html_body:
        return HTMLBody(html_body) 
    return Body(body)
```